core/src/main/java/eu/tsystems/mms/tic/testframework/execution/testng/worker/finish/MethodContextUpdateWorker.java

```diff
@@ -53,8 +53,14 @@ public void onMethodEnd(MethodEndEvent event) {
         // Handle assertions and exceptions in dataprovider methods
         if (testResult.getMethod().isDataDriven()
                 && methodContext.readErrors().anyMatch(ErrorContext::isNotOptional)) {
-            testResult.setStatus(ITestResult.SKIP);
-            methodContext.setStatus(Status.SKIPPED);
+            /**
+             * Skipped is set by a failed data provider method
+             * {@see eu.tsystems.mms.tic.testframework.report.TesterraListener#onDataProviderFailure(ITestNGMethod, ITestContext, RuntimeException)}
```

Comment:
To keep JavaDoc terse, consider
- `import`ing linked classes
- dropping parameters for methods without overloads

```java
/**
 * Skipped is set by a failed data provider method
 * {@see TesterraListener#onDataProviderFailure}
 */
```